New tutorials and scripts

aweinert-MIT opened the pull request:
- Add / update new tutorial formatted as Juypter notebooks
- Added "scripts to rule them all" and used the `em-core` scripts as a template
- Add requirements.txt, so the GitHub dependabot should work now
- README Updates, including distro statement modifications and setup information
- Updated S3 client
- General updates to repo organization and directory structures

aweinert-MIT commented on Tutorials/Get_Started.md:
```diff
@@ -166,4 +167,4 @@ plt.imshow(im, cmap='Greys_r')
 
-[BSD -Clause License](https://github.com/LADI-Dataset/ladi-tutorial/blob/master/LICENSE)
\ No newline at end of file
+[BSD -Clause License](https://github.com/LADI-Dataset/ladi-tutorial/blob/master/LICENSE)
```
Add more detail and reference back to MIT disto language

dnadia commented on scripts/README.md:
```diff
@@ -0,0 +1,157 @@
+The scripts need to be run in a Linux shell. For Windows 10 users, you can use [Ubuntu on Windows](https://tutorials.ubuntu.com/tutorial/tutorial-ubuntu-on-windows#0). Specifically for Windows users, system drive and other connected drives are exposed in the `/mnt/` directory. For example, you can access the Windows C: drive via `cd /mnt/c`.
+
+If you modify these scripts, please follow the [convention guide](https://github.com/Airspace-Encounter-Models/em-overview/blob/master/CONTRIBUTING.md#convention-guide) that specifies an end of line character of `LF (\n)`. If the end of line character is changed to `CRLF (\r)`, you will get an error like this:
+
```
The referenced error message is missing

aweinert-MIT replied:
We used the [`em-core` convention guide](https://github.com/Airspace-Encounter-Models/em-overview/blob/master/CONTRIBUTING.md#convention-guide) as a template when drafting the language here. This is a good catch and this README should reference the [`ladi-overview` convention guide](https://github.com/LADI-Dataset/ladi-overview/blob/master/CONTRIBUTING.md)

aweinert-MIT replied:
Resolved with commit 8f1d064

dnadia commented on Tutorials/README.md:
```diff
@@ -1,6 +1,65 @@
+
+[Scripts](./scripts)
+
+This is a set of boilerplate scripts describing the normalized script pattern that GitHub uses in its projects. The GitHub Scripts To Rule Them All was used as a template. These scripts will download all of the necessary data to get you started.
```
Not relevant to this README, should be moved out to the parent folder (whose README already contains something similar).

aweinert-MIT replied:
Can you update the README to address your comment?

aweinert-MIT replied:
Addressed by 16e329b as part of initial setup instructions

dnadia commented on scripts/README.md:
```diff
@@ -0,0 +1,157 @@
+    - [script/setup](#scriptsetup)
+      - [Data](#data)
+
+## `FLOOD_ANALYSIS_CORE` and Execution
+
+These scripts assume that `FLOOD_ANALYSIS_CORE` has been set. Refer to the repository root [README](../README.md) for instructions.
```
There are no such instructions in the repository root README.

aweinert-MIT replied:
I'll address this

aweinert-MIT replied:
This should had been addressed by commit 16e329b

dnadia commented on scripts/README.md:
````diff
@@ -0,0 +1,157 @@
+
+Depending on your security policy, you may need to run some scripts as a superuser or another user. These scripts have been tested using [`sudo`](https://manpages.ubuntu.com/manpages/disco/en/man8/sudo.8.html). Depending on how you set up the system variable, `FLOOD_ANALYSIS_CORE` you may need to call [sudo with the `-E` flag](https://stackoverflow.com/a/8633575/363829), preserve env.
+
+If running without administrator or sudo access, try running these scripts using `bash`, such as 
+
+```bash
+bash ./setup.sh
+```
````
This is unclear.

aweinert-MIT replied:
You can either execute a script directly (e.g. `./setup.sh` or within a specific shell environment (`bash ./setup.sh`). On enterprise systems, user permissions can vary across environments. On your personal computer, you're less likely to encounter situations like this.

aweinert-MIT commented:
@LADI-Dataset/mit-elo, my last few commits up to 0737343 reorganized the repo, cleaned up documentation, and fixed any broken image links. Also in 8f1d064, I changed the system environment variable name to `LADI_DIR_TUTORIAL`.
